examples/ucp_client_server.c

```diff
@@ -267,43 +452,47 @@ static ucs_status_t request_wait(ucp_worker_h ucp_worker, void *request,
 }
 
 static int request_finalize(ucp_worker_h ucp_worker, test_req_t *request,
-                            test_req_t *ctx, int is_server, void *msg,
-                            int current_iter)
+                            test_req_t *ctx, void *msg, int current_iter)
 {
     ucs_status_t status;
+    data_meta_t *mdata = msg;
     char *msg_str;
+    int rst = 0;
```

Comment:
The variables with initialization should be defined before those without initialization.